app/models/team.rb

```diff
@@ -275,26 +275,10 @@ def custom_permissions(ability = nil)
     perms
   end
 
-  def teamwide_tags
-    self.tag_texts.where(teamwide: true)
-  end
-
-  def custom_tags
-    self.tag_texts.where(teamwide: false)
-  end
-
   def permissions_info
     YAML.load(ERB.new(File.read("#{Rails.root}/config/permission_info.yml")).result)
   end
 
-  def used_tags
-    self.custom_tags.map(&:text)
-  end
-
-  def get_suggested_tags
```

Comment:
You can remove that from the `TeamType` too

Reply:
Missed this, thanks! Done